package.json

```diff
@@ -41,8 +42,10 @@
     "js-yaml": "^3.14.1",
     "lodash.throttle": "^4.0.8",
     "panini": "^1.7.1",
+    "qrcode": "^1.4.4",
     "rimraf": "^3.0.2",
     "slick-carousel": "^1.8.1",
+    "uint8-to-base64": "^0.2.0",
```

Comment:
Please check for licensing, MIT is good

Reply:
ISC

Source:
https://www.npmjs.com/package/uint8-to-base64